dist/tools/git/git-cache

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
```

Comment:
why are you not just setting `#!/bin/sh -ex` here instead of disabling/enabling it for all functions?